3rdparty/breakpad/CMakeLists.txt

```diff
@@ -72,14 +71,20 @@ if(WIN32)
 endif()
 
 
-include_directories(.)
 # ADD_DEFINITIONS( -fPIC )
 add_library(crashreporter-breakpad STATIC ${breakpadSources} )
+target_include_directories( crashreporter-breakpad PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>)
+
+if(WIN32)
+    target_compile_definitions(crashreporter-breakpad PRIVATE UNICODE _UNICODE)
+    if (NOT MSVC)
+        target_compile_options(crashreporter-breakpad PRIVATE -fshort-wchar )
+        target_link_libraries(crashreporter-breakpad PUBLIC mingwex)
+    endif()
+endif()
 
 if(NOT MSVC)
     target_link_libraries(crashreporter-breakpad pthread)
 endif()
 
-if(WIN32 AND NOT MSVC)
-    target_link_libraries(crashreporter-breakpad "mingwex")
-endif()
+add_library(PkgConfig::BREAKPAD ALIAS crashreporter-breakpad)
```

Comment:
I think you should move this line and the definition of the custom target into the `if(WIN32)`. There is no option you can set to link to the custom `breakpad` on non-Windows anyway.